src/js/components/Diagram/README.md

```diff
@@ -68,6 +109,24 @@ svg
 ```
 ## Theme
   
+**diagram.animation**
+
+The animation configuration for Diagram. Expects `object`.
+
+Defaults to
+
+```
+{
+      duration: '1s',
```

Comment:
Why do we need this `duration` property within Diagram? Is it actually being used?
If it is used we should consider using `global.animation.duration` instead. any reason you considered separating it from the global value?

Reply:
I originally thought if we'd put the `duration` property within Diagram, a user might want to only affect just the duration of all the Diagram's they use. After giving it more thought I think it does seem redundant to put a base duration for Diagram and instead make it available at `global.animation.duration`.